recipes/confu_json/all/conanfile.py

```diff
@@ -43,30 +43,24 @@ def _compilers_minimum_version(self):
         }.get(self._min_cppstd, {})
 
     def layout(self):
-        basic_layout(self, src_folder="src")
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.81.0")
-        self.requires("magic_enum/0.8.2")
+        self.requires("boost/1.84.0")
+        self.requires("magic_enum/[>=0.9.5 <10]")
```

Comment:
While we're always on the lookout for more libraries to add, for now we only allow a small subset listed in https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges for version ranges, as those need careful consideration. Could you please pin this to the latest magic_enum release instead? Thanks!